File: Completed/js/alphabetSubSequence/alphabetSubSequence.js
function alphabetSubsequence(str) {
    for (let i = 0; i < str.length - 1; i++) {
        if (str[i] >= str[i + 1]) {
            return false;
        }
    }
    return true
}
console.log(alphabetSubsequence('zab'))
console.log(alphabetSubsequence('effg'))
console.log(alphabetSubsequence('cdce'))
console.log(alphabetSubsequence('ace'))
console.log(alphabetSubsequence('bxz'))

File: Completed/js/alphabetSubSequence/alphabetSubSequence.py
def alphabetSubsequence(s):
    for i in range(len(s)-1):
        if(s[i] >= s[i+1]):
            return False
    return True


print(alphabetSubsequence('zab'))
print(alphabetSubsequence('effg'))
print(alphabetSubsequence('cdce'))
print(alphabetSubsequence('ace'))
print(alphabetSubsequence('bxz'))
